### PR TITLE
RAB-1130: Replace box/spout by openspout/openspout

### PR DESCRIPTION
**Pull request by ghoussard**
**Description (for Contributor and Core Developer)**

In this PR, I replaced `box/spout` by `openspout/openspout`. That's the dependency we use to deal with CSV/XLSX file reading. Unfortunately, it is [not maintained anymore](https://github.com/box/spout#-archived-project-) by Box team. A [fork](https://github.com/openspout/openspout#openspout) has been created and is still maintained under `openspout/openspout`.

Manual testing:
I done CSV and XLSX export/import and Tailored Import on envs with box/spout and openspout/openspout, I didn't see any regression on behavior or performance.

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -108,6 +107,7 @@
         "monolog/monolog": "1.25.5",
         "ocramius/proxy-manager": "2.11.1",
         "oneup/flysystem-bundle": "^4.0.0",
+        "openspout/openspout": "^3.3.0",
```
Comment (ghoussard) [inline composer.json:110]: A v4 exists and an upgrade will be done in another PR

**Review by StevenVAIDIE — APPROVED** [review]

**Review by ValentinMumble — APPROVED** [review]
